source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOAdaptiveCardParseResult.h

```diff
@@ -5,12 +5,14 @@
 //  Copyright © 2017 Microsoft. All rights reserved.
 //
 
-#import "ACOParseResult.h"
+#import <Foundation/Foundation.h>
 
 @class ACOAdaptiveCard;
 
-@interface ACOAdaptiveCardParseResult:ACOParseResult
+@interface ACOAdaptiveCardParseResult:NSObject
 
 @property ACOAdaptiveCard *card;
+@property BOOL IsValid;
```

Comment:
>IsValid [](start = 15, length = 7)

`isValid`, with a lowecase i, to conform to normal naming conventions.

You might also consider making all these fields readonly (and immutable in the case of parseErrors), since there's no reason for callers to modify these objects.